### PR TITLE
fix obsolete docker info

### PR DESCRIPTION
**Pull request by tengpeng**

### CODE REVIEW

File: doc/build.md [hunk 1]
```diff
@@ -276,8 +276,12 @@ These are updated on a weekly basis with the latest builds of MXNet. Examples of
 are as follows:
 
 ```bash
+# CPU only docker
 sudo docker run -it kaixhin/mxnet
-sudo docker run -it --device /dev/nvidiactl --device /dev/nvidia-uvm --device /dev/nvidia0 kaixhin/cuda-mxnet:7.0
+
+OR
+# CPU enabled docker
```
Comment (piiswrong) [inline doc/build.md]: GPU enabled docker?

Comment (mli) [inline doc/build.md]: docker with gpu is quite straightforward with https://github.com/NVIDIA/nvidia-docker

**Comment by tengpeng** [comment]
@piiswrong  Sorry for the typo.

**Comment by tengpeng** [comment]
@mli I have tested the nvidia docker. I passed sample tests and its digits set. However, I fail to get the mxnet docker works.

**Pull request merged**